server/src/main/java/org/elasticsearch/action/admin/indices/template/get/GetIndexTemplatesRequest.java

```diff
@@ -34,6 +34,7 @@ public class GetIndexTemplatesRequest extends MasterNodeReadRequest<GetIndexTemp
     private String[] names;
 
     public GetIndexTemplatesRequest() {
+        this.names = new String[0];
```

Comment:
Maybe a comment to explain, what's happening in this case?
Do we get templates for all indices?

Reply:
Yes,  the RequestBuilder did this implicitly due to varargs usage. Using this constructor without calling .indices resulted in a NPE.

I guess an alternative would be to remove this ctor